### DISCUSSION TITLE
How to include "Activity" column in generated TimeSheet pdf ?

### DISCUSSION THREAD

**Discussion by ProfaBG** [kimai/kimai#4462] [Questions & Answers]
Hi all,

I am trying to figure out how to include column Activity when generating timesheet reports.
I've read on documentation and searched the issues but did not find anything similar.

So far i've tried messing with timesheet.pdf.twig (templates/exports/redered) as well as with index.html.twig and pdf-layout.html.twig - always resulting in breaking of my hosted app.

Is this possible without custom fields addin?

**Comment by kevinpapst** [comment]
Yes it is.
https://github.com/kimai/kimai/blob/main/templates/export/renderer/timesheet.pdf.twig#L14

**Reply by ProfaBG** [reply]
Thank you!
I went trough this and i have no idea what to do =/ 
Im am very very very new to this and am learning as i go. 

What kind of modification would i need to do to this file to include activity column?

I've tried editing this section, but it just breks my app:
{% elseif entry.activity is not null %}
        {{ entry.activity.name }}
        
I know these type of questions must be very basic, even stupid, but believe me i just found out about selfhosting few weeks ago and im not really any kind of developer. 

I really appreciate any kind of help and pointers!

ND

**Reply by kevinpapst** [reply]
This contains the activity name: `{{ entry.activity.name }}`

**Comment by ProfaBG** [comment]
i will figure it out
